hypermap/aggregator/tasks.py

```diff
@@ -81,6 +90,43 @@ def check_layer(self, layer):
         task_error.save()
 
 
+@shared_task(bind=True)
+def index_cached_layers(self):
+    """
+    Index all layers in the Django cache (Index all layers who have been checked).
+    """
+    from hypermap.aggregator.models import Layer
+    from hypermap.aggregator.solr import SolrHypermap
+    from hypermap.aggregator.models import TaskError
```

Comment:
we have this line in models:
from tasks import update_endpoint, update_endpoints, check_service, check_layer

not sure if we can easily remove the circular import. I will look into it at a later stage.